app/partials/states/event_detail.html

```diff
@@ -79,126 +79,19 @@ <h1>{{event.title}}</h1>
-              <strong>Dietary Requirements or Relevant Medical Conditions</strong>
-              <span aria-haspopup="true" class="icon-help has-tip" data-ot="For example, it is important for us to know if you have a severe allergy and/or carry an EpiPen, are prone to fainting, suffer from epilepsy..."></span>
-            </label>
-            <input id="medical-reqs" name="medical-reqs" type="text" ng-model="additionalInformation.medicalRequirements">
-
-            <label for="access-reqs">
-              <strong>Accessibility Requirements</strong>
-              <span aria-haspopup="true" class="icon-help has-tip" data-ot="For example, please let us know if you need wheelchair access, hearing loop or if we can help with any special adjustments."></span>
-            </label>
-            <input id="access-reqs" name="access-reqs" type="text" ng-model="additionalInformation.accessibilityRequirements">
-
-            <div ng-show="additionalInformation.yearGroup != 'TEACHER' && additionalInformation.yearGroup != 'OTHER'" class="row">
-              <div class="columns small-16">
-                <h3>Emergency Contact Details</h3>
-              </div>             
-              <div class="columns small-16 medium-8">            
-                <label for="emergency-name">
-                  <strong>Contact Name</strong>
-                  <span class="ru-form-required"></span>
-                </label>
-                <input id="emergency-name" name="emergency-name" ng-model="additionalInformation.emergencyName" type="text" required/>
-              </div>
-              <div class="columns small-16 medium-8">            
-                <label for="emergency-number">
-                  <strong>Contact Telephone Number</strong>
-                  <span class="ru-form-required"></span>
-                </label>
-                <input id="emergency-number" name="emergency-number" type="text" ng-model="additionalInformation.emergencyNumber" required/>
-              </div>
-            </div>
-
-          </fieldset>
+            <div event-booking-form ></div>
+      
           <span ng-if="!event.expired">
-            <span ng-if="user._id && event.numberOfPlaces > 0 && !event.userBooked && !bookingDeadlinePast && (event.placesAvailable > 0 || (event.tags.indexOf('student') != -1 && user.role != 'STUDENT'))">
+            <span ng-if="user._id && event.numberOfPlaces > 0 && !event.userBooked && !bookingDeadlinePast && event.eventStatus != 'WAITING_LIST_ONLY' && (event.placesAvailable > 0 || (event.tags.indexOf('student') != -1 && user.role != 'STUDENT'))">
               <p><small>By requesting to book on this event, you are granting event organisers access to the information provided in the form above. You are also giving them permission to set you pre-event work and view your progress.  You can manage access to your progress data in your <a ui-sref="accountSettings">account settings</a>.</small></p>
```

Comment:
Minor, but I would say that we probably need to include this statement if event.eventStatus == 'WAITING_LIST_ONLY'.